### ISSUE TITLE
Key and values with hashes

### ISSUE THREAD

**Issue by Blueblau** [webdeveric/webpack-assets-manifest#1]
I'm getting both keys and values with hashes. 

```
{
  "static/images/bag.png": "static/images/bag.9fadf04.png",
  "static/images/intro-1.jpg": "static/images/intro-1.b5ec5ab.jpg",
  "static/images/intro-2.jpg": "static/images/intro-2.c219b15.jpg",
  "static/images/intro-3.jpg": "static/images/intro-3.4114be0.jpg",
  "static/images/intro-4.jpg": "static/images/intro-4.62cad73.jpg",
  "static/images/intro-5.jpg": "static/images/intro-5.e5b599b.jpg",
  "static/images/wheel.png": "static/images/wheel.e52b322.png",
  "static/images/onboarding-1.png": "static/images/onboarding-1.c95e4e0.png",
  "static/images/onboarding-2.png": "static/images/onboarding-2.46027c1.png",
  "static/images/onboarding-3.png": "static/images/onboarding-3.69cfa9a.png",
  "static/images/onboarding-4.png": "static/images/onboarding-4.3dfdc5a.png",
  "static/images/quizHeader.png": "static/images/quizHeader.4a1e428.png",
  "static/images/graveinput.png": "static/images/graveinput.194c07e.png",
  "static/images/crash.png": "static/images/crash.4e3778a.png",
  "static/images/lostLife.png": "static/images/lostLife.daf7ffd.png",
  "static/images/tombstone.png": "static/images/tombstone.5ab9e88.png",
  "static/images/died.png": "static/images/died.9cacdd1.png",
  "static/images/filled-bag.png": "static/images/filled-bag.da7cf71.png",
  "manifest.e6c2c23167923a80c7b8.js": "static/js/manifest.e6c2c23167923a80c7b8.js",
  "manifest.js.map": "static/js/manifest.e6c2c23167923a80c7b8.js.map",
  "vendor.8087a05f98669c87a8a7.js": "static/js/vendor.8087a05f98669c87a8a7.js",
  "vendor.js.map": "static/js/vendor.8087a05f98669c87a8a7.js.map",
  "app.a25e1b4cd2d0619f5344.js": "static/js/app.a25e1b4cd2d0619f5344.js",
  "app.a502dac04e73f2196e1beb24090ad205.css": "static/css/app.a502dac04e73f2196e1beb24090ad205.css",
  "app.js.map": "static/js/app.a25e1b4cd2d0619f5344.js.map",
  "app.css.map": "static/css/app.a502dac04e73f2196e1beb24090ad205.css.map"
}
```

My setup is:
```
- /src
      - images
      - js 
- /static (built by webpack, by asset-folder, with hashes)
- /dist (copied static with hashes)
```

**Comment by webdeveric** [comment]
@Blueblau Can you post your webpack config?

**Comment by webdeveric** [comment]
@Blueblau I'm closing this due to no reply in three weeks.

I've released v0.2.0 which has better file extension handling. Please update and try again.

If you're still having issues, please let me know.
